app/views/hyrax/my/works/_list_works.html.erb

```diff
@@ -28,7 +28,15 @@
       </div>
     </div>
   </td>
-  <td class="date text-center"><%= document.date_modified %></td>
+  <td class="date text-center">
+    <% if document.date_modified.nil? %>
```

Comment:
I think the reason something like this wasn't done before is that date_modified and date_uploaded are core fields and are set by the system, not the user.  I wonder if you can put just the change to this file in a separate PR.  Everything else looks good to go.  I'd like to think about this change.